go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/google/certificate-transparency-go
 go 1.16
 
 require (
+	bitbucket.org/creachadair/shell v0.0.6 // indirect
```

Comment:
Not a showstopper, but any idea how this came in?!

Reply:
I popped it in since the build broke without it (it was one of the go 1.16 missing indirect things)